c2cgeoportal/scaffolds/create/vars_+package+.yaml_tmpl

```diff
@@ -59,6 +59,8 @@ vars:
 
     layers:
         geometry_validation: True
+        last_update_date_column: False
+        last_update_user_column: False
```

Comment:
yes, it could be moved to the UI metadata but that's not a UI feature, only server side